addon/components/vo-mu-file-dropzone.js

```diff
@@ -28,9 +28,12 @@ export default class VoMuFileDropzoneComponent extends Component {
   }
 
   get endPoint() {
-    return this.args.endPoint || '/file-service/files';
+    return this.args.endPoint || '/files';
```

Comment:
perhaps best to default to the old default for backwards compatibility

Reply:
discussed with felix, the new default makes more sense. we'll release a major release